multimodal-search-tirg/pods/image-read.yml

```diff
@@ -0,0 +1,22 @@
+!ImageReader
+with:
+  channel_axis: -1
+requests:
+  on:
+    IndexRequest:
+      - !CraftDriver {}
+    SearchRequest:
+      - !FilterQL
+        with:
+          traversal_paths: ['c']
+          lookups: {'modality': 'image'}
```

Comment:
I am not sure, would check if this `lookup` would work or we need to do some `modality__eq` or `modality__in`. So double check